test/integration/apiserver/admissionwebhook/load_balance_test.go

```diff
@@ -58,154 +65,191 @@ func TestWebhookLoadBalance(t *testing.T) {
 		t.Fatalf("Failed to build cert with error: %+v", err)
```

Comment:
if I recall correctly, we had issues with this test flaking when it was first introduced. Did you run it a bunch of times in a row to check that this reworked version is stable?

Reply:
You mean `tls.X509KeyPair()` here or the whole test, which I'd guess the connection count is not always predictable. 

Anyway, I ran more than a dozen times in a raw and no failures.

Reply:
sorry, the whole test, just had to pick a random line to comment on

Reply:
FTR, @aojea suggested `stress` following https://gist.github.com/liggitt/6a3a2217fa5f846b52519acfc0ffece0#running-integration-tests-to-reproduce-flakes

```
stress ./admissionwebhook.test -test.run TestWebhookLoadBalance
5s: 0 runs so far, 0 failures
10s: 0 runs so far, 0 failures
15s: 0 runs so far, 0 failures
20s: 0 runs so far, 0 failures
25s: 0 runs so far, 0 failures
30s: 0 runs so far, 0 failures
35s: 3 runs so far, 0 failures
40s: 11 runs so far, 0 failures
45s: 16 runs so far, 0 failures
50s: 16 runs so far, 0 failures
55s: 16 runs so far, 0 failures
1m0s: 16 runs so far, 0 failures
1m5s: 20 runs so far, 0 failures
1m10s: 24 runs so far, 0 failures
1m15s: 31 runs so far, 0 failures
1m20s: 32 runs so far, 0 failures
1m25s: 32 runs so far, 0 failures
1m30s: 32 runs so far, 0 failures
1m35s: 36 runs so far, 0 failures
1m40s: 41 runs so far, 0 failures
1m45s: 46 runs so far, 0 failures
1m50s: 48 runs so far, 0 failures
1m55s: 48 runs so far, 0 failures
2m0s: 48 runs so far, 0 failures
2m5s: 48 runs so far, 0 failures
2m10s: 48 runs so far, 0 failures
2m15s: 50 runs so far, 0 failures
2m20s: 53 runs so far, 0 failures
2m25s: 57 runs so far, 0 failures
2m30s: 60 runs so far, 0 failures
2m35s: 63 runs so far, 0 failures
2m40s: 64 runs so far, 0 failures
2m45s: 64 runs so far, 0 failures
2m50s: 67 runs so far, 0 failures
2m55s: 71 runs so far, 0 failures
3m0s: 75 runs so far, 0 failures
3m5s: 78 runs so far, 0 failures
3m10s: 80 runs so far, 0 failures
3m15s: 80 runs so far, 0 failures
3m20s: 82 runs so far, 0 failures
3m25s: 84 runs so far, 0 failures
3m30s: 89 runs so far, 0 failures
3m35s: 92 runs so far, 0 failures
3m40s: 95 runs so far, 0 failures
3m45s: 96 runs so far, 0 failures
3m50s: 98 runs so far, 0 failures
3m55s: 99 runs so far, 0 failures
4m0s: 104 runs so far, 0 failures
4m5s: 108 runs so far, 0 failures
4m10s: 110 runs so far, 0 failures
4m15s: 112 runs so far, 0 failures
```